packages/components/src/components/hds/app-header/index.hbs

```diff
@@ -3,7 +3,12 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<div class={{this.classNames}} ...attributes>
+<div
+  class={{this.classNames}}
+  {{! @glint-expect-error - https://github.com/josemarluedke/ember-focus-trap/issues/86 }}
+  {{focus-trap isActive=this.shouldTrapFocus}}
```

Comment:
Since @alex-ju approved, I added this focus trap to keep focus within the AppHeader when the menu is expanded in mobile view. Could someone else please sign off on these latest changes since Alex will be out for awhile?

Reply:
Tested this out. Very nice!! 👍